cdc/capture/capture.go

```diff
@@ -114,13 +115,19 @@ type captureImpl struct {
 	newOwner func(upstreamManager *upstream.Manager) owner.Owner
 }
 
+// disableDrain is a boolean that disable capture drain feature.
+// Because TiCDC release-6.2 does not support cross version upgrade and stops
+// service during upgrade, it is impossible to perform graceful shutdown.
+const disableDrain = true
```

Comment:
Seems it's not configurable.

Reply:
Yes, because this PR targets on release-6.2 branch where the best effort graceful shutdown is impossible during upgrading.